ods_ci/tests/Resources/OCP.resource

```diff
@@ -95,6 +95,17 @@ Fetch CA Certificate If RHODS Is Self-Managed
         Fetch Openshift CA Bundle
     END
 
+Get Events And Pod Logs
+    [Documentation]    Fetches the namespace events and the pod logs (all containers).
+    ...                It prints and return both items
+    [Arguments]    ${namespace}    ${label_selector}
+    ${rc}    ${events}=    Run And Return Rc And Output    oc get events -n ${namespace}
+    Log    ${events}
+    ${rc}    ${podlogs}=    Run And Return Rc And Output
+    ...    oc logs -l ${label_selector} --all-containers -n ${namespace}
+    Log    ${podlogs}
```

Comment:
I don't know if it's a good idea to print all these logs, as they might be very long. IMHO it would be better to write them to a file.

Reply:
it is done in this way to we can see them soon instead of going finding files, besides we should handle the filenames as well to avoid overwriting.
I believe it's worth to try having the logs here. If it really helps and we find suboptimal to have all the logs printed here we can give another round. Too many changes in this PR for now IMO

Reply:
>  the logs overall less readable.


the log don't get visible unless you open the `Log` statement in the HTML


> For me it's ok to try this approach ad re-evaluate down the line though


okay, thx!